src/app/config/staticContent.ts

```diff
@@ -0,0 +1,152 @@
+                iconSource:
+                    'https://pixabay.com/pl/vectors/%c5%bcar%c3%b3wka-pomys%c5%82-roz%c5%bcarzony-lampa-1294335/',
+                href: 'https://www.nasa.gov/feature/facts-and-figures',
+                content: 'There were over 140 spaceflight launches last year',
+            },
+            {
+                icon: fact,
+                iconClassName: 'icon-small',
+                alt: 'Light bulb icon',
+                iconSource:
+                    'https://pixabay.com/pl/vectors/%c5%bcar%c3%b3wka-pomys%c5%82-roz%c5%bcarzony-lampa-1294335/',
+                href: 'https://www.nasa.gov/feature/facts-and-figures',
+                content:
+                    'There are at least 7 people on space station right now',
+            },
+            {
+                icon: fact,
+                iconClassName: 'icon-small',
+                alt: 'Light bulb icon',
+                iconSource:
+                    'https://pixabay.com/pl/vectors/%c5%bcar%c3%b3wka-pomys%c5%82-roz%c5%bcarzony-lampa-1294335/',
+                href: 'https://www.nasa.gov/feature/facts-and-figures',
+                content:
+                    'In 24 hours, the space station is travelling through 16 sunrises and sunsets',
+            },
+            {
+                icon: fact,
+                iconClassName: 'icon-small',
+                alt: 'Light bulb icon',
+                iconSource:
+                    'https://pixabay.com/pl/vectors/%c5%bcar%c3%b3wka-pomys%c5%82-roz%c5%bcarzony-lampa-1294335/',
+                href: 'https://www.nasa.gov/feature/facts-and-figures',
+                content:
+                    'The space for living and working on space station is larger than a six-bedroom house',
+            },
+        ],
+        contentTypes: [
+            {
+                icon: news,
+                iconClassName: 'icon-small',
+                alt: 'Newspaper icon',
+                iconSource:
+                    'https://pixabay.com/pl/vectors/gazeta-czyta%c4%87-aktualno%c5%9bci-biznes-311272/',
+                href: null,
+                content: 'Latest spaceflight news',
+            },
+            {
+                icon: blog,
+                iconClassName: 'icon-small',
+                alt: 'Sheet of paper with fountain pen icon',
+                iconSource:
+                    'https://pixabay.com/pl/vectors/pisa%c4%87-list-poczta-d%c5%82ugopis-tekst-1292838/',
+                href: null,
+                content: 'Detailed blogs about missions and launches',
+            },
+            {
+                icon: reports,
+                iconClassName: 'icon-medium',
+                alt: 'Graph on monitor icon',
+                iconSource:
+                    'https://pixabay.com/pl/vectors/p%c5%82aski-projekt-symbol-ikona-www-2126876/',
+                href: null,
+                content: 'Latest data from space stations and missions',
+            },
```

Comment:
it's good practice to do not leave a trailing comma in the end of the list. I think prettier could be configured to keep that rule.

Reply:
done